cmd/exporter.go

```diff
@@ -77,10 +77,14 @@ func main() {
 	if err != nil {
 		klog.Fatalf("%s", fmt.Sprintf("failed to create collector: %v", err))
 	}
-	err = newCollector.Attach()
-	if err != nil {
+	_ = newCollector.Attach()
+	/*
+		remove this part of error check for now, as it binds with bcc library
```

Comment:
@rootfs , could you please suggest/advise?

Reply:
https://github.com/sustainable-computing-io/kepler/actions/workflows/unit_test_mac.yml
https://github.com/sustainable-computing-io/kepler/actions/workflows/unit_test.yml
for those two jobs we don't have bcc.

the reason in deep for the failure is about the reader logic.
https://github.com/sustainable-computing-io/kepler/blob/main/pkg/collector/reader.go#L106-L109
and some null pointer return by `pkg/attacher/bcc_attacher_stub.go` if bcc not installed.

I suppose there are two ways, either we make our code robust(for rest api handler part) or we make our code decouple with bcc(if we can't decouple with bcc, we force ourself bind with it.)

Reply:
@SamYuan1990 if this is doable, maybe have yet another bcc attacher stub implementation for integration test and in the stub function, return certain mock data
https://github.com/sustainable-computing-io/kepler/blob/main/pkg/attacher/bcc_attacher_stub.go#L42-L52
e.g. 
```go
func (table *Table) Iter() *TableIterator {
	return &TableIterator{mock_data}
}

func (it *TableIterator) Next() bool {
	return false
}

func (it *TableIterator) Leaf() []byte {
	return mock_data
}
```